int32 id
spatial_db_msgs/Pose3DModel model
---

